napari/layers/points/points.py

```diff
@@ -250,7 +250,9 @@ def __init__(
             size=Event,
             edge_width=Event,
             face_color=Event,
```

Comment:
why did you create the `current_face_color` event instead of re-using the `face_color` event?

Reply:
I added the current_* events to update the current value in the layer control dropdowns. Previously, the face_color and edge_color dropdown menus were triggered by the face_color event and updated from `Points.current_face_color` and `Points.current_edge_color`. `face_color` and `current_face_color` aren't always updated at the same time, so I thought it made sense to create an event specifically for the property that is being viewed. Parts of the view are refreshed by the `face_color` event, so thought it was better not to refresh them every time current_face_color is updated (e.g., in add mode and selecting a new color for the next point to be added). Do you think that makes sense?

Reply:
yeah it does, we might want to separate them for all `current_*` properties.